plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/GcpLocalRunTab.java

```diff
@@ -200,14 +203,16 @@ public void modifyText(ModifyEvent event) {
     });
     Button browse = new Button(composite, SWT.NONE);
     browse.setText(Messages.getString("button.browse")); //$NON-NLS-1$
+    String[] filterExtensions = new String[] {"*.json"}; //$NON-NLS-1$
```

Comment:
Does `GOOGLE_APPLICATION_CREDENTIALS` support P12 files?  I know we can download them from Pantheon.

Reply:
My gut feeling is that it supports P12 files or such. However, the assumption here based on my discussion with @elharo is that, we'll need only service accounts (`.json`) for the Cloud libraries.

Reply:
P12 is old and busted. JSON is the new hotness:

![image](https://user-images.githubusercontent.com/1005544/32960689-ed401ec0-cb93-11e7-9097-eee9a4cb574e.png)